deploy/examples/client/client-secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakClient
+metadata:
+  name: client-secret
+spec:
+  realmSelector:
+    matchExact: master
+  instanceSelector:
```

Comment:
seeing this beside the `realmSelector`, this might be better changed to `keycloakSekector`. However, since we are only targetting single namespace support for now, this can be removed and added later once multi namespace support is added. What do you think?

Reply:
I would actually leave it as is. I think the multi-namespace support will become a priority very soon.

Reply:
sounds good to me